x/xbasm/parse.go

```diff
@@ -39,3 +39,169 @@ func ParseFnCallClaims(v basm.MarshaledAttestedObject) (*FnCallClaims, error) {
 
 	return c, nil
 }
+
+func isNonZero(b []byte) bool {
```

Comment:
All code below here may look familar, it is the `[][]byte` decoder from delphi.  I am putting this in here for this release but am working on something that makes it easier to manage moving forward.